src/internal/staking.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * This file contains the underlying implementations for exposed API surface in
+ * the {@link api/staking}. By moving the methods out into a separate file,
+ * other namespaces and processes can access these methods without depending on the entire
+ * faucet namespace and without having a dependency cycle error.
+ */
+
+import { AptosConfig } from "../api/aptos_config";
+import { Hex } from "../core";
+import { GetDelegatedStakingActivitiesResponse, GetNumberOfDelegatorsResponse, HexInput, OrderBy } from "../types";
+import { GetDelegatedStakingActivitiesQuery, GetNumberOfDelegatorsQuery } from "../types/generated/operations";
+import { GetDelegatedStakingActivities, GetNumberOfDelegators } from "../types/generated/queries";
+import { queryIndexer } from "./general";
+
+export async function getNumberOfDelegators(args: {
```

Comment:
I would rename here to `getNumberOfDelegatorsByPoolAddress` and the second one `getNumberOfDelegators`